modules/serverless-sinkbinding-odc.adoc

```diff
@@ -0,0 +1,109 @@
+.Procedure
+
+. Create a Knative service to use as a sink:
+.. In the *Developer* perspective, navigate to *+Add* -> *YAML*.
+.. Copy the example YAML:
++
+[source,yaml]
+----
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: event-display
+spec:
+  template:
+    spec:
+      containers:
+        - image: quay.io/openshift-knative/knative-eventing-sources-event-display:latest
+----
+.. Click *Create*.
+
+. Create a `CronJob` resource that is used as an event source and sends an event every minute.
+.. In the *Developer* perspective, navigate to *+Add* -> *YAML*.
+.. Copy the example YAML:
++
+[source,yaml]
+----
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: heartbeat-cron
+spec:
+  # Run every minute
+  schedule: "*/1 * * * *"
+  jobTemplate:
+    metadata:
+      labels:
+        app: heartbeat-cron
+        bindings.knative.dev/include: true <1>
+    spec:
+      template:
+        spec:
+          restartPolicy: Never
+          containers:
+            - name: single-heartbeat
+              image: quay.io/openshift-knative/heartbeats
+              args:
+              - --period=1
+              env:
+                - name: ONE_SHOT
+                  value: "true"
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: POD_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
+----
+<1> Ensure that you include the `bindings.knative.dev/include: true` label. The default namespace selection behavior of {ServerlessProductName} uses inclusion.
+.. Click *Create*.
+
+. Create a sink binding in the same namespace as the service created in the previous step, or any other sink that you want to send events to.
+
+.. In the *Developer* perspective, navigate to *+Add* -> *Event Source*. The  *Event Sources* page is displayed.
+.. Optional: If you have multiple providers for your event sources, select the required provider from the *Providers* list to filter the available event sources from the provider.
+.. Select *Sink Binding* and then click *Create Event Source*. The *Create Event Source* page is displayed.
+.. In the *apiVersion* field enter `batch/v1`.
+.. In the *Kind* field enter `Job`.
++
+[NOTE]
+====
+The `CronJob` kind is not supported directly by {ServerlessProductName} sink binding, so the *Kind* field must target the `Job` objects created by the cron job, rather than the cron job object itself.
+====
+.. Select a *Sink*. This can be either a *Resource* or a *URI*. In this example, the `event-display` service created in the previous step is used as the *Resource* sink.
+.. In the *Match labels* section:
+... Enter `app` in the *Name* field.
+... Enter `heartbeat-cron` in the *Value* field.
++
+[NOTE]
+====
+The label selector is required because {ServerlessProductName} only supports job selection by labels, not names.
+====
```

Comment:
It's also required because Jobs created by CronJob have a random string in their name, like `hearthbeat-cron-1cc23f`. So, even if ODC provided a way to specify a name (see [ODC-6341](https://issues.redhat.com/browse/ODC-6341)), we still need to use here the label based.

Reply:
BTW. match labels are on every ODC version. None of them have name field.

Reply:
Decided to just mention the random string bit instead of calling out something unsupported - this makes more sense as a reason to use labels anyway.